webapp/lib/view.dart

```diff
@@ -364,6 +364,10 @@ class ConversationPanelView {
     _conversationWarning.title = '';
     _conversationWarning.classes.add('hidden');
   }
+
+  void scrollToLatest() {
+    _messages.scrollTop = _messages.scrollHeight;
```

Comment:
if I remember correctly, there is a `scrollIntoView` method (or something like that) on HTML elements. Could we potentially use that directly on the tag element, if it behave as expected?

Reply:
I've tied using ScrollIntoView but it doesn't seem to be working as expected.